site/cleanup.md

```diff
@@ -1,21 +1,28 @@
 # Cleanup
 
-The Image Tool creates a temporary directory, prefixed by ```wlsimgbuilder_temp```,
-every time it runs.  Under normal circumstances, this directory will be deleted. However,
-if the process is aborted or the tool is unable to remove the directory, you can delete it manually.
+The Image Tool creates a temporary Docker context directory, prefixed by ```wlsimgbuilder_temp```,
+every time the tool runs.  Under normal circumstances, this context directory will be deleted. However,
+if the process is aborted or the tool is unable to remove the directory, it is safe for you to delete it manually.
 
-By default, the temporary directory is created under the user's home directory. If you do not want it created under the home directory, set the environment variable:
+By default, the Image Tool creates the Docker context directory under the user's home directory. 
+If you prefer to use a different directory for the temporary context, set the environment variable `WLSIMG_BLDDIR`.
 
  ```bash
 export WLSIMG_BLDDIR="/path/to/dir"
 ```
 
 
-If you see dangling images after the build, use the following command to remove them:
+The Image Tool will try to prune intermediate images from the Docker multi-stage build after the build step. 
+If you see dangling images after the build (images labeled as `<none>`, use the following command to remove them:
```

Comment:
build (images labeled as `<none>`, -> build (images labeled as `<none>`),   (added the missing closing parenthesis)